src/Field/Definition/ChoiceField.php

```diff
@@ -72,8 +72,8 @@ protected function toManagementApiData () : array
 				"default_value" => $this->defaultValue instanceof \BackedEnum
 					? $this->defaultValue->value
 					: $this->defaultValue,
-				"min_options" => $this->minimumNumberOfOptions,
-				"max_options" => $this->maximumNumberOfOptions,
+				"min_options" => is_int($this->minimumNumberOfOptions) ? "{$this->minimumNumberOfOptions}" : null,
+				"max_options" => is_int($this->minimumNumberOfOptions) ? "{$this->maximumNumberOfOptions}" : null,
```

Comment:
```suggestion
				"min_options" => null !== $this->minimumNumberOfOptions)
					? (string) $this->minimumNumberOfOptions 
					: null,
				"max_options" => null !== $this->minimumNumberOfOptions 
					? (string) $this->maximumNumberOfOptions 
					: null,
```

Always check for null and then just cast instead of doing string interpolation

Reply:
We should also add comment above these lines, why we do these changes.